ext/rbs_extension/constants.c

```diff
@@ -1,34 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* This file is generated by the templates/template.rb script and should not  */
+/* be modified manually.                                                      */
+/* To change the template see                                                 */
+/* templates/ext/rbs_extension/constants.c.erb                                */
+/*----------------------------------------------------------------------------*/
+
 #include "rbs_extension.h"
 
 VALUE RBS_Parser;
 
 VALUE RBS;
 VALUE RBS_AST;
-VALUE RBS_AST_Comment;
-VALUE RBS_AST_Annotation;
-VALUE RBS_AST_TypeParam;
-
 VALUE RBS_AST_Declarations;
+VALUE RBS_AST_Directives;
+VALUE RBS_AST_Members;
+VALUE RBS_Parser;
+VALUE RBS_Types;
+VALUE RBS_Types_Bases;
```

Comment:
These are special-cased in the template. The rest of the file is code-genned, but in a different order than the original. All the same constants are still here, though.